rtb/analisi-requisiti/src/casi-uso.d/1-utente-autenticato.tex

```diff
@@ -179,25 +180,27 @@ \subsubsection{UC8-Cancellazione profilo}
 \end{itemize}
 
 \subsubsection{UC9-Selezione profilo cliente}
-\begin{figure}[h] \includegraphics[scale=1]{uc09.png} \end{figure}
+\begin{figure}[h] \includegraphics[scale=1]{uc09.png} \end{figure} 
 \begin{itemize}
+\item \textbf{Descrizione:} Un account può possedere più profili al suo interno. I profili sono identificati tramite il loro nome profilo e l'account a cui appartengono. Il loro funzionamento è analogo a quello un comune servizio di streaming. % quest'ultima frase rende l'idea ma non sono sicuro se sia corretto metterla qui
```

Comment:
A mio parere non è il luogo corretto, io sposterei questa frase (che di per sé è corretta e rende bene l'idea) nella sezione "Attori", in particolare nel punto "Utente autenticato".